meson.build

```diff
@@ -186,6 +186,36 @@ if not get_option('snpe-support').disabled()
   endif
 endif
 
+# tensorrt
+if not get_option('tensorrt-support').disabled()
+  # check available cuda versions (11.0 and 10.2 are recommended)
+  cuda_vers = [
+    '11.0',
+    '10.2',
+    '10.1',
+    '10.0',
+    '9.2',
+    '9.1',
+    '9.0'
```

Comment:
- as far as I know, if the tensorRT was compiled with a different version of cuda and cudnn, even though a different version of cuda/cudnn is available, it won't work (this is from what I experienced an year ago).
if this happens currently as well, I am not sure if these cuda version versions will help unless there is the specific cuda version check.
- as long as version checks are being added, cudnn version might also be helpful

Reply:
That's right. Any installed TensorRT expects the exact version of cuda/cudnn. I will change the PR.

Reply:
Hmm... Unfortunately, it's really hard (impossible) to find the corresponding cuda version from libnvinfer. Do you have idea?

Reply:
I cant think of any way either.